sample-apps/react-native/dogfood/src/screens/LiveStream/HostLiveStream.tsx

```diff
@@ -127,6 +127,14 @@ export const HostLiveStreamScreen = ({ route }: HostLiveStreamScreenProps) => {
     getOrCreateCall();
   }, [call, connectedUser]);
 
+  const CustomLiveStreamMediaControls = useCallback(() => {
+    return (
+      <LivestreamMediaControls
+        handlePresentModalPress={handlePresentModalPress}
+      />
+    );
+  }, [handlePresentModalPress]);
```

Comment:
`handlePresentModalPress` could also be created inside this callback

Reply:
Done ✅